resources/views/friend_profile.blade.php

```diff
@@ -70,22 +70,7 @@
             @endif
 
             {{-- フレンド解除ボタン --}}
-            {{-- TODO:まだフレンドじゃないときはフレンド登録ボタンにする --}}
-            
-            {{-- @foreach ($me->friends as $friend)
-            @if($friend->friend_id == $user->id)
-            {{Form::open(['route'=>['friend_delete',$user->id]])}}
-            {{Form::submit('フレンド解除')}}
-            {{Form::close()}}
-            @elseif($friend->friend_id != $user->id || )
-            {{Form::open(['route'=>'add_friend'])}}
-            {{Form::hidden('friend_id',$user->id)}}
-            {{Form::submit('フレンドになる',['name'=>'friend_btn'])}}
-            {{Form::close()}}
-            @endif
-            @endforeach --}}
-            @foreach ($user->friends as $friend)
-            @if($friend->user_id ==Auth::id())
+            @if($is_friend->count() > 0)
```

Comment:
`is_friend` を existsで取得すると型がbooleanになるので下記のように判定出来ると思います
```suggestion
            @if ($is_friend)
```